app/user/[id]/page.tsx

```diff
@@ -422,7 +466,7 @@ export default function UserDashboard() {
                   {!hasInitiallyLoaded ? (
                     <div className="h-7 bg-gray-200 dark:bg-gray-700 rounded animate-pulse"></div>
                   ) : (
```

Comment:
A tiny UX improvement here would be to show a shimmer instead of the hyphen in order to unify our stats card loading visuals.

The Total Work stat depends on `accountData` which loads separately from `userData`. Without this check, the shimmer disappears once `hasInitiallyLoaded` becomes true, but `Total Work` still shows "-" while `accountData` finishes loading.

Adding `|| isLoadingAccountData` ensures all stats show shimmer until their respective data sources have loaded, creating a more consistent loading experience.
```suggestion
                  {!hasInitiallyLoaded || isLoadingAccountData ? (
                    <div className="h-7 bg-gray-200 dark:bg-gray-700 rounded animate-pulse"></div>
                  ) : (
```